docs/website/docs/HardWonWisdom.md

```diff
@@ -112,3 +112,196 @@ REQ2: If L≤k (see REQ1) and L<R, we must validate at least L+1 of their header
+But that's just an ideal case; we could relax that, if we had a reason to embrace the additional complexity it involves.
+The actual fundamental requirement derived from Praos is two-fold.
+
+- No false alarms for honest headers: an honest header (which is by definition actually valid) must never appear as invalid.
+
+- Never too many missed alarms: no adversary can mint a chain of apparently-valid headers that has `k` or more headers after its intersection with any honest chain within the stability window.
+  (TODO Is this constraint overly-precise?)
+
+- (Compare that pair to the "Why use the Honest Chain Growth window as the Ledger's Stability Window?" question in `HardWonWisdom.md`.)
+
+Roughly: the rule is to not (at all) underestimate the honest chain growth and to not (TODO "significantly" or "at all"?) over-estimate the adversarial chain growth.
+
+Consider some examples.
+
+- Even in this laxer rule, the real `d` parameter cannot change within the stability window, because if it does change, then honest headers that are in an overlay slot according to the reduced `d` might not be in an overlay slot according to the unreduced `d`, and that would lead to false negatives for honest (overlay) headers.
+
+- It would be possible to have defined the overlay schedule such the overlaidness of a slot was monotonic in `d`, even though we didn't do it that way.
+  Even if we had, then there still would have been false alarms for honest headers, because some honest headers in slots that were actually not overlaid would be rejected since the incorrectly greater `d` would interpret those slots as overlaid.
+  So the `d` parameter is something that even the laxer rule could not accomodate (even when assuming it can't increase, because the its overall influence on honest headers is not monotonic).
+
+- As of epoch 257, `d=0` (ie no slots are overlaid).
+  Since then, no change to the Cardano protocol parameters (including era transitions) has changed the (average) number of slots any given issuer leads during any given interval of slots.
+  Also since then, every change that has at all affected the validity of headers has been an era transition.
+  Therefore, _none of those subsequent changes truly required a stability window of warning_.
+  Honest headers after such change would correctly identify themselves as such by claiming the new era, and therefore would be correctly validated.
+  An adversarial header, in this hypothetical, could have falsely claimed to be from the new era, but the crucial insight is that doing so wouldn't have (TODO "significantly" or "at all"?) increased the chance the adversary could create an apparently-valid chain that is long enough to cause problems.
+
+Recall that that discussion was merely theoretical, to better delineate the fundamental requirements.
```

Comment:
I think it would make sense to specify upfront what invariant that is present in the current design/implementation, but which you are temporarily discarding here for the (very useful!) sake of exploring the design space, i.e. "assume the safe zone is empty".